src/macros.jl

```diff
@@ -343,6 +346,12 @@ Jacobi (generic function with 1 method)
 
 julia> Jacobi(11,35)
 1
```

Comment:
To also test the `GapInt` auto-conversion could try this:
```suggestion
julia> Jacobi(11,35)
1

julia> Jacobi(big(35)^100+11, 35)
1
```
But not urgent.